### PR TITLE
Use new Storage API in main index.ts

### PR DESCRIPTION
**Pull request by glennhartmann**
ServiceWorker later.

Note that this is a fairly mechanical replacement for now,
and we'll probably want to rethink/refactor index.ts more,
but for now at least this shows that the new API is working.

(depends on https://github.com/GoogleChromeLabs/llaminator/pull/42 for now, can be rebased if we decide against that)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -27,81 +28,59 @@ if ('serviceWorker' in navigator && process.env.NODE_ENV !== 'development') {
...
-    console.log('put success:', e)
-  });
-  t.addEventListener('error', (e) => {
-    console.log('put error:', e)
-    // TODO: display error
-  });
+  const recordReady = db.add(b, {
+    filename: f.name,
+    mimeType: f.type,
+    // title: '',
```
Comment (beverloo) [inline src/index.ts:66]: nit: s/b/blob/
nit: s/f/file/

We're using the full names on :82 and :83, and while I indeed prefer descriptive variable names, we should at the very least be consistent at the file-level until we figure this out through linting.

File: src/storage.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ interface LlamaDB extends DBSchema {
   };
 }
 
-export default class LlamaStorage {
+export class LlamaStorage {
```
Comment (beverloo) [inline src/storage.ts:55]: 😃

File: src/index.ts [hunk 3]
```diff
@@ -18,6 +18,7 @@ import '@material/mwc-top-app-bar-fixed';
 
 import './components/llama-select-fab';
 import './llaminator.css';
+import * as storage from './storage';
```
Comment (beverloo) [inline src/index.ts:21]: nit:
```typescript
import { LlamaStorage, mainImageName } from './storage';
```

This is where the `k` prefix would really clarify _what_ it is, as opposed to a free function.

**Review by beverloo — APPROVED** [review]
SG, few nits but they're all redundant with previous PRs.

File: src/index.ts (hunk 3 above)
Comment (glennhartmann) [reply src/index.ts]: done.

File: src/index.ts (hunk 1 above)
Comment (glennhartmann) [reply src/index.ts:66]: done.

**Pull request merged**